cli/src/semgrep/scan_report.py

```diff
@@ -39,9 +39,7 @@ def _print_product_status(sast_enabled: bool = True, sca_enabled: bool = False)
     (Simple) print the statuses of enabled products to stdout when the user
     is given the product-focused CLI UX treatment.
     """
-    learn_more_url = with_color(
-        Colors.cyan, "https://semgrep.dev/products/cloud-platform/", underline=True
-    )
+    learn_more_url = with_color(Colors.cyan, "https://sg.run/cloud", underline=True)
```

Comment:
Why do we do that? Why adding an intermediate, that we need to configure, when plain regular URLs are working?
Also what is the difference between https://semgrep.dev/explore and registry.semgrep.dev? (just followed sg.run which had a link on this new registry.semgrep.dev)

Reply:
> Why adding an intermediate, that we need to configure, when plain regular URLs are working?

There are a few advantages that this layer of indirection provides, namely for the benefit of our product marketing team (see [slack thread](https://semgrepinc.slack.com/archives/C02JT6YA5S7/p1707862629618569)), and currently this change does not introduce any new functionality.

I'm also partial to the [fundamental theorem of software engineering](https://en.wikipedia.org/wiki/Fundamental_theorem_of_software_engineering) pearl of wisdom where "We can solve any problem by introducing an extra level of [indirection](https://en.wikipedia.org/wiki/Indirection)."

Reply:
> Also what is the difference between https://semgrep.dev/explore and registry.semgrep.dev?

The `explore` page is the home of our current rules registry, whereas `registry.semgrep.dev` is an experimental flavor of the registry that Drew built.